src/components/header.en.js

```diff
@@ -0,0 +1,92 @@
+import React from "react";
+import styled from "@emotion/styled";
+import { css } from "@emotion/core";
+import { Link, graphql, useStaticQuery } from "gatsby";
+import { Trans } from "@lingui/macro";
+import { I18nProvider } from "@lingui/react";
+import catalogEN from "../locales/en/messages";
+import catalogES from "../locales/es/messages";
+
+const NavLink = styled(Link)`
+  color: #eeeeee;
+  font-size: 1.2rem;
+  font-weight: ${props => props.fontWeight || "normal"};
+  line-height: 1;
+  margin: 0 0.5rem 0 0;
+  padding: 0.25rem;
+  text-decoration: none;
+  color: #ffffff;
+  text-shadow: 3px 5px 2px #474747;
+  &.current-page {
+    border-bottom: 2px solid #ffffff;
+  }
+
+  &:last-of-type {
+    margin-right: 0;
+  }
+`;
+
+const Header = () => {
+  const { image } = useStaticQuery(graphql`
+    query {
+      image: file(relativePath: { eq: "air4childrenLogo1.png" }) {
```

Comment:
Maybe you can update the logo with the latest version available here https://github.com/air4children/logos

Reply:
oh nice. I updated the logo [here](https://github.com/air4children/site/pull/5/commits/8736298340f859e3d5b4310c3e047a795a6dc78e), but I am not using any images by now.

Reply:
Thanks Angel, for pointed out that. It is good news as logo will still change : )